meta.go

```diff
@@ -859,6 +955,22 @@ var storageSchema = schema.FieldMap(
 	},
 )
 
+var deviceSchema = schema.FieldMap(
+	schema.Fields{
+		"description": schema.String(),
+		"type": schema.OneOf(
+			schema.Const(string(DeviceGPU)),
+		),
+		"countmin": schema.Int(),
```

Comment:
Let's copy off how storage does this using a schema (see storageCountC)
That means also that the parseDevices () method doesn't need to return an error as the checks are done by the schema